### ISSUE TITLE
model skipping some of the captions

### ISSUE THREAD

**Issue by AbdulrahmanSoliman1** [m87-labs/moondream#122]
Hi 
![image](https://github.com/user-attachments/assets/1a9a6916-aa3a-42df-851f-b12e4222f1f9)
although the train/loss is low
the model always focuses on one part of the captions and ignores the rest. I am using the default fine-tuning
the dataset is 20k images-captions

Ground Truth: Medical image shows broken leg and lighting could impact visibility.
Moondream:  Medical image shows broken leg.

is there something to do that makes the model focus on the whole label 

    def __getitem__(self, idx):
        sample = self.data[idx]
        image_path = os.path.join(self.image_dir, sample["image_file"]) 
        image = Image.open(image_path).convert("RGB")
        return {
            "image": image,
            "qa": [
                {
                    "question": "Describe the following image in detail",
                    "answer": sample["description"],
                }
            ]
        }
